config/environments/production.rb

```diff
@@ -42,8 +42,11 @@
   # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
   # config.assume_ssl = true
 
-  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+# Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies. 
+#If the variable is set to 'true', SSL enforcement will be enabled; otherwise, it will be disabled.
+config.force_ssl = ENV.fetch("FORCE_SSL", "true") == "true"
```

Comment:
Should the default value not a return a boolean value instead of a string?

```ruby
ENV.fetch("FORCE_SSL", true) == "true"
```

If there's a better option than using `"true"` as the value to look for in the environment variable I would use that but I cannot think of one that isn't overly complex.